tests/mock_tables/asic1/ip_special_recursive_route.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file can be removed ?

Reply:
For the test command, it queries from all namespaces and it is needed to indicate there is no route from this particular namespace to see how it is handled.  So this is needed.